### ISSUE TITLE
Cannot build 1.11.0 due to missing module

### ISSUE THREAD

**Issue by undu** [be5invis/Iosevka#145]
* Your font version: 1.11.0
* Your font variant: release/custom
* Your operating system (name and version): archlinux
If you have problem when building Iosevka, please provide these informations:

* Your Node.js version: v7.4.0
* Your results of `npm list`
```
iosevka@1.9.2 /home/pau/dev/aur/ttf-iosevka-source/src/Iosevka-1.11.0
├── bezier-js@2.2.2
├─┬ caryll-shapeops@0.2.0
│ └── clipper-lib@1.0.0
├── cubic2quad@1.1.0
├── libspiro-js@0.3.1
├── otfcc-c2q@0.5.0
├─┬ patel@0.32.0
│ ├─┬ escodegen@1.8.1
│ │ ├── esprima@2.7.3
│ │ ├── estraverse@1.9.3
│ │ ├── esutils@2.0.2
│ │ ├─┬ optionator@0.8.2
│ │ │ ├── deep-is@0.1.3
│ │ │ ├── fast-levenshtein@2.0.6
│ │ │ ├── levn@0.3.0
│ │ │ ├── prelude-ls@1.1.2
│ │ │ ├── type-check@0.3.2
│ │ │ └── wordwrap@1.0.0
│ │ └─┬ source-map@0.2.0
│ │   └── amdefine@1.0.1
│ ├─┬ esmangle@1.0.1
│ │ ├── escodegen@1.3.3
│ │ ├─┬ escope@1.0.3
│ │ │ └── estraverse@2.0.0
│ │ ├── esprima@1.1.1
│ │ ├─┬ esshorten@1.1.1
│ │ │ └── estraverse@4.1.1
│ │ ├── estraverse@1.5.1
│ │ ├── esutils@1.0.0
│ │ ├─┬ optionator@0.3.0
│ │ │ ├── fast-levenshtein@1.0.7
│ │ │ ├── levn@0.2.5
│ │ │ └── wordwrap@0.0.3
│ │ └── source-map@0.1.43
│ ├─┬ patrisika@0.21.0
│ │ └── patrisika-scopes@0.11.0
│ └─┬ resumer@0.0.0
│   └── through@2.3.8
├── toml@2.3.1
├── topsort@0.0.2
├── unorm@1.4.1
└─┬ yargs@6.6.0
  ├── camelcase@3.0.0
  ├─┬ cliui@3.2.0
  │ ├─┬ strip-ansi@3.0.1
  │ │ └── ansi-regex@2.1.1
  │ └── wrap-ansi@2.1.0
  ├── decamelize@1.2.0
  ├── get-caller-file@1.0.2
  ├─┬ os-locale@1.4.0
  │ └─┬ lcid@1.0.0
  │   └── invert-kv@1.0.0
  ├─┬ read-pkg-up@1.0.1
  │ ├─┬ find-up@1.1.2
  │ │ ├── path-exists@2.1.0
  │ │ └─┬ pinkie-promise@2.0.1
  │ │   └── pinkie@2.0.4
  │ └─┬ read-pkg@1.1.0
  │   ├─┬ load-json-file@1.1.0
  │   │ ├── graceful-fs@4.1.11
  │   │ ├─┬ parse-json@2.2.0
  │   │ │ └─┬ error-ex@1.3.0
  │   │ │   └── is-arrayish@0.2.1
  │   │ ├── pify@2.3.0
  │   │ └─┬ strip-bom@2.0.0
  │   │   └── is-utf8@0.2.1
  │   ├─┬ normalize-package-data@2.3.5
  │   │ ├── hosted-git-info@2.1.5
  │   │ ├─┬ is-builtin-module@1.0.0
  │   │ │ └── builtin-modules@1.1.1
  │   │ ├── semver@5.3.0
  │   │ └─┬ validate-npm-package-license@3.0.1
  │   │   ├─┬ spdx-correct@1.0.2
  │   │   │ └── spdx-license-ids@1.2.2
  │   │   └── spdx-expression-parse@1.0.4
  │   └── path-type@1.1.0
  ├── require-directory@2.1.1
  ├── require-main-filename@1.0.1
  ├── set-blocking@2.0.0
  ├─┬ string-width@1.0.2
  │ ├── code-point-at@1.1.0
  │ └─┬ is-fullwidth-code-point@1.0.0
  │   └── number-is-nan@1.0.1
  ├── which-module@1.0.0
  ├── y18n@3.2.1
  └── yargs-parser@4.2.1
```
* Your _otfcc_ version: 0.6.3
* Your FontForge version: None
* Your `parameters.toml`:  default

---

When building the fonts I get an error: "Cannot find module './toPoly'"
```
$ make release
node ./node_modules/patel/bin/patel-c --optimize --strict support/glyph.ptl -o support/glyph.js
node ./node_modules/patel/bin/patel-c --optimize --strict support/spiroexpand.ptl -o support/spiroexpand.js
node ./node_modules/patel/bin/patel-c --optimize --strict support/spirokit.ptl -o support/spirokit.js
node ./node_modules/patel/bin/patel-c --optimize --strict parameters.ptl -o parameters.js
node ./node_modules/patel/bin/patel-c --optimize --strict support/anchor.ptl -o support/anchor.js
node ./node_modules/patel/bin/patel-c --optimize --strict support/point.ptl -o support/point.js
node ./node_modules/patel/bin/patel-c --optimize --strict support/transform.ptl -o support/transform.js
node ./node_modules/patel/bin/patel-c --optimize --strict support/utils.ptl -o support/utils.js
node ./node_modules/patel/bin/patel-c --optimize --strict meta/aesthetics.ptl -o meta/aesthetics.js
node ./node_modules/patel/bin/patel-c --optimize --strict meta/naming.ptl -o meta/naming.js
node ./node_modules/patel/bin/patel-c --optimize --strict meta/features.ptl -o meta/features.js
node ./node_modules/patel/bin/patel-c --optimize --strict glyphs/common-shapes.ptl -o glyphs/common-shapes.js
node ./node_modules/patel/bin/patel-c --optimize --strict glyphs/overmarks.ptl -o glyphs/overmarks.js
node ./node_modules/patel/bin/patel-c --optimize --strict glyphs/letters-unified-basic.ptl -o glyphs/letters-unified-basic.js
node ./node_modules/patel/bin/patel-c --optimize --strict glyphs/letters-unified-extended.ptl -o glyphs/letters-unified-extended.js
node ./node_modules/patel/bin/patel-c --optimize --strict glyphs/numbers.ptl -o glyphs/numbers.js
node ./node_modules/patel/bin/patel-c --optimize --strict glyphs/symbol-punctuation.ptl -o glyphs/symbol-punctuation.js
node ./node_modules/patel/bin/patel-c --optimize --strict glyphs/symbol-math.ptl -o glyphs/symbol-math.js
node ./node_modules/patel/bin/patel-c --optimize --strict glyphs/symbol-letter.ptl -o glyphs/symbol-letter.js
node ./node_modules/patel/bin/patel-c --optimize --strict glyphs/symbol-geometric.ptl -o glyphs/symbol-geometric.js
node ./node_modules/patel/bin/patel-c --optimize --strict glyphs/symbol-other.ptl -o glyphs/symbol-other.js
node ./node_modules/patel/bin/patel-c --optimize --strict glyphs/symbol-braille.ptl -o glyphs/symbol-braille.js
node ./node_modules/patel/bin/patel-c --optimize --strict glyphs/autobuilds.ptl -o glyphs/autobuilds.js
node ./node_modules/patel/bin/patel-c --optimize --strict buildglyphs.ptl -o buildglyphs.js
Building iosevka-thin with iosevka w-thin s-upright
Building iosevka-thinitalic with iosevka w-thin s-italic
Building iosevka-thinoblique with iosevka w-thin s-oblique
Building iosevka-extralight with iosevka w-extralight s-upright
Building iosevka-extralightitalic with iosevka w-extralight s-italic
Building iosevka-extralightoblique with iosevka w-extralight s-oblique
Building iosevka-light with iosevka w-light s-upright
Building iosevka-lightitalic with iosevka w-light s-italic
module.js:472
    throw err;
    ^

Error: Cannot find module './toPoly'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/pau/dev/aur/ttf-iosevka-source/src/Iosevka-1.11.0/node_modules/caryll-shapeops/index.js:10:14)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
make: *** [build/targets.mk:30: build/iosevka-thinitalic.0.otd] Error 1
make: *** Waiting for unfinished jobs....
module.js:472
    throw err;
    ^

Error: Cannot find module './toPoly'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/pau/dev/aur/ttf-iosevka-source/src/Iosevka-1.11.0/node_modules/caryll-shapeops/index.js:10:14)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
make: *** [build/targets.mk:102: build/iosevka-extralightoblique.0.otd] Error 1
module.js:472
    throw err;
    ^

Error: Cannot find module './toPoly'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/pau/dev/aur/ttf-iosevka-source/src/Iosevka-1.11.0/node_modules/caryll-shapeops/index.js:10:14)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
make: *** [build/targets.mk:84: build/iosevka-extralightitalic.0.otd] Error 1
module.js:472
    throw err;
    ^

Error: Cannot find module './toPoly'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/pau/dev/aur/ttf-iosevka-source/src/Iosevka-1.11.0/node_modules/caryll-shapeops/index.js:10:14)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
make: *** [build/targets.mk:12: build/iosevka-thin.0.otd] Error 1
module.js:472
    throw err;
    ^

Error: Cannot find module './toPoly'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/pau/dev/aur/ttf-iosevka-source/src/Iosevka-1.11.0/node_modules/caryll-shapeops/index.js:10:14)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
make: *** [build/targets.mk:66: build/iosevka-extralight.0.otd] Error 1
module.js:472
    throw err;
    ^

Error: Cannot find module './toPoly'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/pau/dev/aur/ttf-iosevka-source/src/Iosevka-1.11.0/node_modules/caryll-shapeops/index.js:10:14)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
module.js:472
    throw err;
    ^

Error: Cannot find module './toPoly'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/pau/dev/aur/ttf-iosevka-source/src/Iosevka-1.11.0/node_modules/caryll-shapeops/index.js:10:14)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
make: *** [build/targets.mk:120: build/iosevka-light.0.otd] Error 1
make: *** [build/targets.mk:48: build/iosevka-thinoblique.0.otd] Error 1
module.js:472
    throw err;
    ^

Error: Cannot find module './toPoly'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/pau/dev/aur/ttf-iosevka-source/src/Iosevka-1.11.0/node_modules/caryll-shapeops/index.js:10:14)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
make: *** [build/targets.mk:138: build/iosevka-lightitalic.0.otd] Error 1
```

**Comment by AluisioASG** [comment]
Got the same issue a while ago; made a fix at caryll/shapeops#1.

**Comment by be5invis** [comment]
Update your deps with `npm update` will fix this.
Closed.
